link/src/main/java/com/stripe/android/link/ui/wallet/PaymentDetails.kt

```diff
@@ -118,16 +121,31 @@ internal fun PaymentDetailsListItem(
                 )
             }
         }
-        IconButton(
-            onClick = onMenuButtonClick,
-            modifier = Modifier.padding(end = 6.dp),
-            enabled = enabled
+
+        Box(
+            contentAlignment = Alignment.Center,
+            modifier = Modifier
+                .size(MinimumTouchTargetSize)
+                .padding(end = 12.dp)
         ) {
```

Comment:
Wrapped this in a `Box` to make sizing consistent. Also added more end padding, so that they menu icon is aligned with the collapsed/expanded icon of the header.